contracts/libraries/Commands.sol

```diff
@@ -38,37 +38,15 @@ library Commands {
     // The following constant defines one of the boundaries where the if blocks split commands
     uint256 constant SECOND_IF_BOUNDARY = 0x10;
 
-    // Command Types where 0x10<=value<0x18, executed in the third nested-if block
-    uint256 constant SEAPORT_V1_5 = 0x10;
-    uint256 constant LOOKS_RARE_V2 = 0x11;
-    uint256 constant NFTX = 0x12;
-    uint256 constant CRYPTOPUNKS = 0x13;
```

Comment:
These should probably be taken out of the README too. might want to give the README attention in general